app/src/lib/stats/stats-store.ts

```diff
@@ -227,12 +227,14 @@ export class StatsStore {
     this.db = db
     this.uiActivityMonitor = uiActivityMonitor
 
-    this.optOut = getBoolean(StatsOptOutKey, false)
+    const storedValue = getBoolean(StatsOptOutKey)
+
+    this.optOut = storedValue || false
 
     // If the user has set an opt out value but we haven't sent the ping yet,
     // give it a shot now.
     if (!getBoolean(HasSentOptInPingKey, false)) {
-      this.sendOptInStatusPing(!this.optOut)
+      this.sendOptInStatusPing(!this.optOut, storedValue || null)
```

Comment:
Similar question as below, shouldn't this be
```suggestion
      this.sendOptInStatusPing(!this.optOut, storedValue === undefined ? null : storedValue)
```

Or else we'll end up treating `false` as the same thing as the value not being set at all, right?

Reply:
63679a5d82ab97260288c1d22fe385e59a0ff177